webapp/src/editortoolbar.tsx

```diff
@@ -225,6 +231,9 @@ export class EditorToolbar extends data.Component<ISettingsProps, EditorToolbarS
             case "notimpl":
                 modalBody = lf("WebUSB is not supported by this browser; please check for updates.");
                 break;
+            default:
+                modalBody = lf("WebUSB should be supported, but we're having trouble. Please try refreshing the page.");
```

Comment:
```suggestion
                modalBody = lf("Unable to connect to WebUSB. Please try refreshing the page.");
```

Reply:
I worry to promise "should be supported" when we don't know all the details about the user's machine/setup.